### DISCUSSION TITLE
How to protect my connections?

### DISCUSSION THREAD

**Discussion by stefano-p** [iterate-ch/cyberduck#15369] [Q&A]
In my view, cyberduck looks really nice, but it seems an important security feature is totally missing. After I have configured my connections and set my favorite ones, how shall I prevent unauthorized users from launching this tool and access all my data? I would expect at least a kind of a master password or something, so that my configurations are not visible to anyone. If there were a portable version of cyberduck, this would solve this problem. So, my question is: am I missing something? How can I protect my configured connection?

**Comment by dkocher** [comment]
Your connections are secured by storing passwords in the login password store of the operating system (Keychain on macOS and Credentials Manager on Windows) secured by your login password.

**Reply by stefano-p** [reply]
I am afraid that relying on Credential Manager is not satisfactory, in case someone gets access to your computer. That's why other tools offer the possibility to locally secure your data (connections) with a master password. Without such a feature I'll have to skip this tool and find something else. Again, this would be a pity, because all other aspects of Cyberduck are great.

**Comment by stefano-p** [comment]
I would suggest to implement the possibility to add a master password, in a similar way as in S3 Browser.
